engine/lite/k8sDeploymentManager/src/test/scala/pl/touk/nussknacker/k8s/manager/BaseK8sDeploymentManagerTest.scala

```diff
@@ -32,7 +32,7 @@ class BaseK8sDeploymentManagerTest extends AnyFunSuite with Matchers with Extrem
 
   protected lazy val k8s: KubernetesClient = k8sInit
   protected lazy val k8sTestUtils = new K8sTestUtils(k8s)
-  protected val dockerTag = sys.env.getOrElse("dockerTagName", BuildInfo.version)
+  protected val dockerTag = sys.env.getOrElse("dockerTagName", "staging-latest")
```

Comment:
It is temporary, am I right?

Reply:
thanks for the catch. of courese, it is.